packages/@okta/vuepress-site/docs/guides/registration-inline-hook/prepare-your-external-service/nodejs/prepare.md

```diff
@@ -0,0 +1,31 @@
+In this implementation of an external service, the code is written for Node and uses the Express framework. Several npm packages are required:
+
+```javascript
+const express = require('express');
+const bodyParser = require('body-parser');
+```
+
+The following code listens for HTTPS requests:
+
+```javascript
+const app = express();
+const port = process.env.PORT || 3000;
+```
+
+```javascript
+app.listen(port, () => console.log(`Listening on port ${port}!`));
+```
+
+And the following code is required to authorize and parse the call from Okta:
+
+```javascript
+const basicAuth = require('express-basic-auth');
+```
+
+```javascript
+app.use(bodyParser.json());
+app.use(basicAuth({
+  users: { 'admin': 'supersecret' },
+  unauthorizedResponse: req => 'Unauthorized'
+}));
+```
```

Comment:
If you're making this check in the code for the value of the Authorization header, the Hook will only work if you specify the specific encoded value it's looking for as the value of `Authentication secret` when you configure the hook in the Admin Console. Having this check here, but no instructions for configuring the secret during setup, will lead to this not working.

Reply:
Yes ... I can now provide more details in the "Enable and Activate" section where this task is documented, which is now fairly generic. It's a bit of a chicken and egg scenario - both code and admin setup required for it to work. Perhaps, I can be more explicit in the overview about this.